frontend/src/scenes/sceneLogic.ts

```diff
@@ -236,20 +241,37 @@ export const sceneLogic = kea<sceneLogicType>({
         ],
     },
     urlToAction: ({ actions }) => {
+        const unmountFeatFlagLogic = featureFlagLogic.mount()
+        const featFlags = featureFlagLogic.values.featureFlags
+
         const mapping: Record<string, (params: Params) => any> = {}
 
+        let experimentRedirect: string | undefined = undefined
+
         for (const [paths, redirect] of Object.entries(redirects)) {
             for (const path of paths.split('|')) {
+                if (path === '/' && redirect) {
```

Comment:
This doesn't need to run for every redirect path, further the logic here gets convoluted. You just need to check if `path === '/'` and redirect to `/home`. No need to alter this redirect logic.

Reply:
Not sure exactly what you mean here – this logic executes only if path === / 

We have a redirect right now that sets / --> /insights – I don't want to alter this rule or the default behavior. Since we are running this as an experiment I only want a subset of those redirects to go to home

Do you have an idea for an alternate approach?

Reply:
Simplifying logic here in updated commit